query/src/sql/planner/binder/mod.rs

```diff
@@ -114,10 +116,16 @@ impl<'a> Binder {
         catalog_name: &str,
         database_name: &str,
         table_name: &str,
+        travel_point: &Option<TimeTravelPoint>,
```

Comment:
```suggestion
        travel_point: Option<&TimeTravelPoint>,
```

Reply:
oops, there are compilation errors after this suggestion is applied. so I revert it.

travel point is captured here:
https://github.com/datafuselabs/databend/blob/e2a4d789b12805d5d50eda755828ca2c872c2251/common/ast/src/parser/query.rs#L139
Which is of type `Option<TimeTravelPoint>`.

did I missed something here?  

Or  `Option<&T>` style is recommended in new parser/planner?

Reply:
It's recommended, so you don't need to write a `&None`.